utils/fields.py

```diff
@@ -60,7 +61,13 @@ def generate_fake_data(self, faker, **kwargs):
         if (overrides := kwargs.get("overrides")) and self.name in overrides:
             return overrides[self.name]
 
-        return faker.get_fake_data_for_type(self.data_type)
+        params = (
+            [self.definition.get("minimum", 0), self.definition.get("maximum", 9999)]
+            if self.data_type == DataType.INTEGER
+            else []
+        )
+
+        return faker.get_fake_data_for_type(self.data_type, params)
```

Comment:
Would you be willing to add a test for the min/max situation to `test_fields` and/or `test_parsers`? Optional ☺️